doc/source/building/windows.rst

```diff
@@ -168,90 +281,115 @@ the following in your build log:**
 
    FOUND:
       libraries = ['openblas']
-      library_dirs = ['C:\opt\64\lib']
-      language = c
+      library_dirs = ['C:\...........\lib']
+      language = f77
       define_macros = [('HAVE_CBLAS', None)]
 
+We'll get back to this below how to check.
+
 Building SciPy
 ==============
 
-Once you have built OpenBLAS, it's time to build SciPy. Before continuing make sure to
-install the following software for building on the latest Python version. For building
-on other Python versions, see the WindowsCompilers_ page.
+Once you have built OpenBLAS, it's time to build SciPy. Before continuing make
+sure to install the following software for building on the latest Python
+version. For building on other Python versions, see the WindowsCompilers_ page.
+We are also assuming that your Python is on the system path. That is to say,
+when you type `python` in the Windows command prompt the correct Python is
+executed.
+
+Install Microsoft Visual Studio 2017 or 2019 Community Edition (use the
+`build tools`_ from Microsoft). Just like before pick a convenient place to
```

Comment:
What is used from Build Tools? `gfortran`, `gcc`, and even git are from MSYS2, right?

Reply:
Added details.